packages/common/locales/generate-locales-tool/bin/write-locale-files-to-dist.ts

```diff
@@ -6,7 +6,7 @@
  * found in the LICENSE file at https://angular.io/license
  */
 
-import {writeFileSync} from 'fs';
+import fs from 'fs';
```

Comment:
nit:
```suggestion
import {writeFile} from 'fs/promises';
```
Removes the repeat `fs.promises.` below.

Reply:
Yeah, I don’t know what I like more here to be honest 😆I knew about fs/promises

Reply:
I feel like 'fs/promises' is what the rest of the world uses. It was created exactly for this d: